docker-compose.yml

```diff
@@ -332,3 +332,27 @@ services:
 ################################################################################
 # DELTA ORGANIZATIONS: END
 ################################################################################
+
+
+################################################################################
+# MU SEARCH - TODO USE THE CORRECT IMAGE ONCE INHERITANCE IS IN MU SEARCH
+################################################################################
+  search:
+      image: nbittich/mu-search:wip
+      links:
+        - db:database
+      volumes:
+        - ./config/search:/config
+    # environment:
+          # LOG_SCOPE_DELTA: "debug"
+          # LOG_SCOPE_UPDATE_HANDLER: "debug"
+          # LOG_SCOPE_SEARCH: "debug"
+          # LOG_SCOPE_ELASTICSEARCH: "info"
```

Comment:
We could probably remove this ? To keep a docker-compose file as small as possible, I know it tends to grow fast already :sweat_smile: